inference-engine/src/hetero_plugin/hetero_executable_network.cpp

```diff
@@ -1067,6 +1067,30 @@ void HeteroExecutableNetwork::GetMetric(const std::string &name, InferenceEngine
             value = std::max(value, desc._network.GetMetric(METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS)).as<unsigned int>());
         }
         result = IE_SET_METRIC(OPTIMAL_NUMBER_OF_INFER_REQUESTS, value);
+    } else if (name.rfind(METRIC_SUBGRAPH_TAG, 0) == 0) {
```

Comment:
can we remove `METRIC_SUBGRAPH_TAG` as define and use as `"METRIC_SUBGRAPH_TAG"`. I don't want to change public API..

Reply:
just to confirm, line 1070 will become:
} else if (name.rfind("METRIC_SUBGRAPH_TAG", 0) == 0) {

and correspondingly the #define METRIC_SUBGRAPH_TAG TOSTRING(METRIC_SUBGRAPH_) will be removed from ie_plugin_config.hpp at the same time?

Reply:
right, it will be an internal feature.